main.js

```diff
@@ -1,78 +1,61 @@
-const Delegate = require('ftdomdelegate');
 const template = require('./src/js/template');
 const timeoutDuration = 5000;
-const animDuration = 500;
 
-let timeout;
-let currentNotification = null;
+let isInstantiated = false;
+let stack = [];
+let container;
 
-function dispatchNotificationCloseEvent(){
-	document.dispatchEvent(new CustomEvent('nNotification.close'));
-}
+class Notice {
 
-function listenForCloseButtonClick(){
-	let called = false;
-	const func = function(){
-		const delegate = new Delegate(document.body);
-		delegate.on('click', '.n-notification__close-js', hide);
-	};
+	constructor (options) {
 
-	if(!called){
-		func();
-		called = true;
-	}
-}
+		options.trackable = options.trackable || `notification-${options.type}`;
 
-function show(options){
-	clearTimeout(timeout);
+		this.notice = document.createElement('div');
```

Comment:
Is a `div` necessary here? May be good use of a [`new DocumentFragment`](https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment) instead.